src/pages/validator/external-chains/immutable.mdx

```diff
@@ -0,0 +1,337 @@
+Using the WireGuard CLI, create a private and public key for your client.
+
+```bash
+umask 077
+wg genkey > wg-privatekey
+wg pubkey < wg-privatekey > wg-publickey
+```
+
+Using the Axelar CLI, create a transaction with a note that includes a `shasum` hash of your public key.
+
+```bash
+axelard tx bank send broadcaster [any-axelar-address] 1uaxl --note "$( echo axelar-wireguard-[wireguard-pubkey] | shasum -a 256 - )"
+```
+
+Send an email to the [Immutable team](mailto:squad-imx-rollups@immutable.com) including your static IP and
+a link to your transaction in the Axelar explorer. For example:
+
+<div style={{color: 'gray', padding: '1rem'}}>
+> *Please add my public key to the Immutable WireGuard tunnel: `_WIREGUARD_PUBLIC_KEY_`*
+
+> *See my verfication transaction at: `_LINK_TO_EXPLORER_`*
+
+</div>
+
+The Immutable team will reply with a configuration file for your WireGuard client.
+Once you receive this file, update the `PrivateKey` entry with the contents of your `wg-privatekey` file.
+
+Save the WireGuard configuration file in `/etc/wireguard/wg0.conf` and use `wg-quick` to setup the tunnel.
+
+```bash
+wg-quick up wg0
+```
+
+The output should look something like this:
+```bash
+[#] ip link add wg0 type wireguard
+[#] wg setconf wg0 /dev/fd/63
+[#] ip -4 address add <ASSIGNED_CIDR> dev wg0
+[#] ip link set mtu 8921 up dev wg0
+[#] ip -4 route add <GATEWAY_CIDR> dev wg0
+[#] ip -4 route add <GATEWAY_CIDR> dev wg0
+[#] ip -4 route add <GATEWAY_CIDR> dev wg0
+```
+</tab-item>
+</tabs>
+
+### Setup the Immutable zkEVM node
+
+1. Create a data directory.
+
+```bash
+mkdir /opt/immutable-zkevm
+```
+
+2. Generate a random password.
+```bash
+openssl rand -base64 > /opt/immutable-zkevm/password
+```
+
+3. Pull the Docker image.
+
+```bash
+docker pull ghcr.io/immutable/go-ethereum/go-ethereum:v0.0.5
+# Create a short-form tag we can reference in the next few commands
+docker tag ghcr.io/immutable/go-ethereum/go-ethereum:v0.0.5 geth
+```
+
+4. Initialise the node.
+
+<tabs>
+<tab-item title="Testnet">
+```bash
+docker run \
+  --rm \
+  -v /opt/immutable-zkevm:/mnt/geth \
+  --name geth \
+  geth account new --datadir /mnt/geth --password /mnt/geth/password
+```
+
+```bash
+docker run \
+  --rm \
+  -v /opt/immutable-zkevm:/mnt/geth \
+  --name geth \
+  geth init \
+  --datadir /mnt/geth /etc/geth/testnet.json 
+```
+</tab-item>
+<tab-item title="Mainnet">
+```bash
+docker run \
+  --rm \
+  -v /opt/immutable-zkevm:/mnt/geth \
+  --name geth \
+  geth account new --datadir /mnt/geth --password /mnt/geth/password
+```
+
+```bash
+docker run \
+  --rm \
+  -v /opt/immutable-zkevm:/mnt/geth \
+  --name geth \
+  geth init \
+  --datadir /mnt/geth /etc/geth/mainnet.json 
+```
+</tab-item>
+</tabs>
+
+5. Start the Immutable zkEVM node as a service
+
+<tabs>
+<tab-item title="Testnet">
+```bash
+docker run \
+  -d \
+  --restart=always \
+  -v /opt/immutable-zkevm:/mnt/geth \
+  --name geth \
+  -p 8545:8545 \
+  geth \
```

Comment:
small nit but there's a random newline here under testnet that doesn't appear under the mainnet tab (on line 196)

Reply:
Fixed to match